### DISCUSSION TITLE
can tabby open telnet links in web browsers?

### DISCUSSION THREAD

**Discussion by comtack** [Eugeny/tabby#8864] [Q&A]
I need to quite often open "telnet://127.0.0.1:9000" links from a web browser (firefox). Right now as my "url handler" I have putty set up using registry editor in windows like this:

HKLM\SOFTWARE\Classes\telnet\shell\open\command
"C:\Program Files\PuTTY\putty.exe" "%1"

I tried multiple options swapping putty with tabby, adding "open", "run" etc. but nothing works. Has anyone tried this? If tabby could open those session from a link in a new tab, that would be great.

Thanks for any answers

**Comment by ashneilson** [comment]
Hello @comtack 👋 

If PR https://github.com/Eugeny/tabby/pull/9383 is accepted and merged, that should provide the ability to register a protocol handler for `telnet://` which then invokes `tabby` with the proposed `quickConnect` command.
